### ISSUE TITLE
V7 Margin (userChrome.hidden.urlbar_iconbox.label_only)

### ISSUE THREAD

**Issue by thunderstone135** [black7375/Firefox-UI-Fix#560]
### Describe the bug

Hi, I just want to let you know that if I enable `userChrome.hidden.urlbar_iconbox.label_only`, it doesn't respect this 👇

```
    #identity-box[pageproxystate="valid"].notSecureText #identity-icon-box {
      padding-inline-end: 5px; /* Original: 8px */
    }
```
### Screenshots

As evidenced by 👇

![Imgur](https://imgur.com/2PVQh6g.png)

I added this line in userChrome.css 👇

```
#identity-box[pageproxystate="valid"]:is(.chromeUI, .extensionPage, .notSecureText),
#urlbar-label-box {
  margin-inline-end: 5px !important;
}
```

To make it look like this 👇

![Imgur](https://imgur.com/ikQ6EHZ.png)

For a closely detailed comparison, I included a regular link 👇

![Imgur](https://imgur.com/2PVQh6g.png)
![Imgur](https://imgur.com/ikQ6EHZ.png)
![Imgur](https://imgur.com/YEjHdWC.png)

### Expected behavior

The margin should be the same for any kind of link in the URL Bar, which is 5px.

### OS

Windows 11

### Firefox Version

Waterfox G5.1.1

### Distribution

- [X] [Original Lepton](https://github.com/black7375/Firefox-UI-Fix)
- [X] [Lepton's photon style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)
- [X] [Lepton's proton style](https://github.com/black7375/Firefox-UI-Fix/tree/proton-style)

### Theme

- [ ] Light
- [X] Dark
- [ ] Alpenglow (System: Light)
- [ ] Alpenglow (System: Dark)
- [ ] System Default
- [ ] Others: (e.g. [Photon Colors](https://addons.mozilla.org/en-US/firefox/addon/photon-colors/), [Google Chrome Dark](https://addons.mozilla.org/en-US/firefox/addon/google-chrome-dark/), [Fairytale Of Nature](https://addons.mozilla.org/en-US/firefox/addon/fairytale-of-nature/))

### `user.js` setup

<details><summary><strong><code>user.js</code> setup</strong></summary><br>

```javascript
// Your settings
```
</details>

### Additional context

N/A

**Comment by black7375** [comment]
As a result of checking, it works if it is not `userChrome.hidden.urlbar_iconbox.label_only`.
https://github.com/black7375/Firefox-UI-Fix/blob/ebc37342ee6140bd87f9b8b64bb4c0ebdc3b5ef4/src/hidden/_index.scss#L56-L66

Show the iconbox only when security is weak, and adjust padding.

---

tldr; It is intended padding. Thanks to report!!
